libs/core/src/lib/time/time-column/time-column.component.ts

```diff
@@ -277,7 +287,9 @@ export class TimeColumnComponent<K, T extends SelectableViewItem<K> = Selectable
         } else if (KeyUtil.isKeyCode(event, UP_ARROW)) {
             this.scrollUp();
             event.preventDefault();
-        } else if (KeyUtil.isKeyType(event, 'numeric') || KeyUtil.isKeyType(event, 'alphabetical')) {
+        } else if (KeyUtil.isKeyType(event, 'numeric')) {
+            this._numericKeyInputHandler(event);
+        } else if (KeyUtil.isKeyType(event, 'alphabetical')) {
             this._queryKeyDownEvent.next(event.key);
```

Comment:
I don't think this actually does anything when an alphabetical character is pressed, but could be cool to allow the user to switch from AM to PM with a P keypress and vice versa?